truthy-falsy.js

```diff
@@ -0,0 +1,117 @@
+// 1
+const isEmployed = true;
+const isOnVacation = false;
+
+function setAlarm(isEmployed, isOnVacation){
+  return isEmployed && !isOnVacation;
+}
+console.log(setAlarm(isEmployed, isOnVacation));
+
+// 2
+function closeCompare(numberA, numberB, numbersMargin){
+  if (numbersMargin === undefined) numbersMargin = 0;
+  const numbersDiff = Math.abs(numberA - numberB);
+  if (numbersDiff <= numbersMargin) return 0
+  if (numberA < numberB) return -1
+  return 1
+}
+
+// 3
+const or = (a, b) => {
+  if (a == true) return true
+  if (b == true) return true
+  return false
+};
+
+const xor = (a, b) => {
+  if (a == true) {
+    if (b === true) return false
+    else return true
+  }
+  if (b == true) {
+    if (a === true) return false
+    else return true
+  }
+  return false
+};
+
+// 4
+function isTruthy(bool, function1, function2) {
+  if (bool == true) return function1();
+  return function2()
+}
+
+// 5
+function rentalCarCost(days) {
+  if (days >= 3 && days <= 6) {
+    return days * 40 - 20
```

Comment:
You're using `days * 40` multiple times.
Create a variable on top of this function
```
const baseCost = days * 40;
```